src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -139,8 +139,10 @@ public static void NonExistentAction_Throws_ActionNotSupportedException()
     // The client should throw a TimeoutException
     [Fact]
     [OuterLoop]
+    [ActiveIssue(3464)]// Issue is in the corefx repo
     public static void SendTimeout_For_Long_Running_Operation_Throws_TimeoutException()
```

Comment:
Thanks for checking on the different approaches. Yeah, I can't think of a good way to double check that we timed out for the right reason.

I don't think we should close #273 till we've verified the fix, but I don't feel strongly about it.  I've used issues in our repo to track issues in other repo's.  Those links show up when they look at the bug, and seeing ours is Active is a good way to say we're waiting on it.